creusot/src/backend/dependency.rs

```diff
@@ -172,22 +171,17 @@ impl<'tcx> Dependency<'tcx> {
 
     pub(crate) fn base_ident(self, tcx: TyCtxt<'tcx>) -> Symbol {
         match self {
-            Dependency::Type(ty) => {
-                let nm = match ty.kind() {
-                    TyKind::Adt(def, _) => {
-                        item_symb(tcx, def.did(), rustc_hir::def::Namespace::TypeNS)
-                    }
-                    TyKind::Alias(_, aty) => tcx.item_name(aty.def_id),
-                    TyKind::Closure(def_id, _) => {
-                        item_symb(tcx, *def_id, rustc_hir::def::Namespace::TypeNS)
-                    }
-                    _ => Symbol::intern("debug_ty_name"),
-                };
-                Symbol::intern(&nm.as_str().to_snake_case())
-            }
+            Dependency::Type(ty) => match ty.kind() {
+                TyKind::Adt(def, _) => item_symb(tcx, def.did(), rustc_hir::def::Namespace::TypeNS),
+                TyKind::Alias(_, aty) => tcx.item_name(aty.def_id),
+                TyKind::Closure(def_id, _) => {
+                    item_symb(tcx, *def_id, rustc_hir::def::Namespace::TypeNS)
+                }
+                _ => Symbol::intern("debug_ty_name"),
+            },
```

Comment:
Here the identifiers already seem to have been initial-lowercased (as far as the test suite can tell)